assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -80,6 +80,7 @@ function combineElements(state, { firstId, firstElement, secondId }) {
     focalX = 50,
     focalY = 50,
     isFill = false,
+    flip = { vertical: false, horizontal: false },
```

Comment:
Should we perhaps have a constant for these repeating default values?

Reply:
Done in 0994bb0 - though I'm not a huge fan of importing from elements here, I do think it's the best case. Alternatively we'd do this defaulting out where `combineElements` is called, but that's less nice.

Reply:
Found a better solution in 9dfc2d2e (also the other solution didn't work because of circular dependencies).